rts/Lua/LuaConstEngine.cpp

```diff
@@ -31,6 +31,8 @@
  * @field buildFlags string Gets additional engine buildflags, e.g. "Debug" or "Sync-Debug"
  * @field featureSupport FeatureSupport Table containing various engine features as keys; use for cross-version compat
  * @field wordSize number Indicates the build type always 64 these days
+ * @field gameSpeed number Number of FPS, constant 30
```

Comment:
Is this always constant, as documented?

Reply:
I wouldn't put the 30 in the doc because the whole point is to abstract it away.

Reply:
> So currently it is constant to 30 but maybe in the future we'd support showing a different number?

Yes.

>  What is the actual thing we want to expose here, the target ticks per second

It's the target number of ticks (gameframes) per second the server will produce when running at x1 speed.

> Would lower sym speed due to throttling to other players affect this value if we supported it dynamic?

No. For dynamic speed there's `Spring.GetGameSpeed()`, though it's a bit underdocumented (see #1989).

Reply:
Removed the constant information from documentation.